### DISCUSSION TITLE
Missing icons in figma ui kit?

### DISCUSSION THREAD

**Discussion by Sarapetkm** [primefaces/community#2717] [Figma UI Kit]
Hi, my manager got me the Figma UI kit in april. But I'm lacking both components and icons that exist on primevue.org and for the developers in my team. Can I update my Figma-file somehow? Or how do I get access to the stuff thats missing? 

For example, the icons "pi-file-check" and "pi-file-export" are lacking for me, and I need them haha..

Thanks, 
Sara

**Comment by onursenture** [comment]
You can download the latest UI Kit version and follow the tutorial about PrimeIcons on the Getting Started page. This way, you will be able to use all the icons in the latest version of PrimeIcons.

**Comment by MadzMadzik** [comment]
What about Data Table headers that have filtering options? I seem not to be able to find it in Figma.

**Reply by onursenture** [reply]
Some Table features aren’t available in Figma, and Filter is one of them. Including every feature would have added unnecessary complexity and made maintenance harder.
